airqo-platform/AirQo-frontend
# Overflow fix when switched to other languages (Link remainder)

#### Summary of Changes (What does this PR do?)

- Overflow fix when switched to other languages (Link remainder)

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Run the app and switch to a different language to check out if there overflows or not

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
